client/src/components/Login/ForgotPasswordModal.jsx

```diff
@@ -0,0 +1,84 @@
+import {
+  Input,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Checkbox,
+} from '@chakra-ui/react';
+import { FormButton } from 'components/common/Button.jsx';
+import useRegisterForm from 'hooks/useForgotPasswordForm.jsx';
```

Comment:
`useRegisterForm` is not right and misleading.

Reply:
solved, changed to [useForgotPasswordForm]